### PR TITLE
Rewrite config for single domain hosting

### PR DESCRIPTION
**Pull request by yakky**

### CODE REVIEW

File: roles/app/defaults/main.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 rtd_repo: https://github.com/italia/docs.italia.it/
 rtd_italia_repo: https://github.com/yakky/italia_rtd/
-rtd_branch: italia-2.3.11
+rtd_branch: feature/urls
```
Comment (xrmx) [inline roles/app/defaults/main.yml]: not sure you want to commit this

File: roles/app/handlers/main.yml [hunk 2]
```diff
@@ -1,29 +1,5 @@
...
-  service: name=rtd_gunicorn_admin state=restarted
-  listen: "restart rtd"
-
-- name: restart rtd_gunicorn_api
-  service: name=rtd_gunicorn_api state=restarted
-  listen: "restart rtd"
-
-- name: restart rtd_gunicorn_redirect
-  service: name=rtd_gunicorn_redirect state=restarted
+- name: restart rtd
```
Comment (xrmx) [inline roles/app/handlers/main.yml]: Are you losing the ability to restart celery and gunicorn separately?

File: roles/app/templates/nginx/sites-available/readthedocs-main.conf [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+        add_header X-Deity docs;
+    }
+
+    # Let's encrypt
+    location ^~ /.well-known/acme-challenge/ {
+        default_type "text/plain";
+        root /var/www/letsencrypt;
+    }
+
+    ssl_certificate /etc/letsencrypt/live/docs-italia-staging.teamdigitale.it/fullchain.pem; # managed by Certbot
```
Comment (xrmx) [inline roles/app/templates/nginx/sites-available/readthedocs-main.conf]: at least the domain should be a variable no? OTOH If it's managed by certbot we can probably remove it.

File: roles/app/templates/nginx/sites-available/readthedocs-main.conf [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+    ssl_certificate_key /etc/letsencrypt/live/docs-italia-staging.teamdigitale.it/privkey.pem; # managed by Certbot
+}
+
+server {
+    if ($host = docs-italia-staging.teamdigitale.it) {
```
Comment (xrmx) [inline roles/app/templates/nginx/sites-available/readthedocs-main.conf]: same here

Comment (yakky) [reply roles/app/templates/nginx/sites-available/readthedocs-main.conf]: Agree. Fixed

File: roles/app/templates/nginx/sites-available/readthedocs-main.conf (hunk 3 above)
Comment (yakky) [reply roles/app/templates/nginx/sites-available/readthedocs-main.conf]: Agree. Fixed

File: roles/app/handlers/main.yml (hunk 2 above)
Comment (yakky) [reply roles/app/handlers/main.yml]: Agree. Fixed by splitting the handler

File: roles/app/defaults/main.yml (hunk 1 above)
Comment (yakky) [reply roles/app/defaults/main.yml]: Reverted

**Comment by xrmx** [comment]
I've run vagrant config on top of this so at least it runs :)

**Pull request merged**